src/unstract/adapters/llm/palm/src/palm.py

```diff
@@ -48,21 +46,18 @@ def get_json_schema() -> str:
 
     def get_llm_instance(self) -> Optional[LLM]:
         try:
-            if self.config.get(Constants.NUM_OUTPUT) is not None:
-                llm = PaLM(
-                    model=str(self.config.get(Constants.MODEL)),
-                    api_key=str(self.config.get(Constants.API_KEY)),
-                    num_output=int(self.config.get(Constants.NUM_OUTPUT, 50)),
-                    api_type=Constants.API_TYPE,
-                    temperature=0,
-                )
-            else:
-                llm = PaLM(
-                    model=str(self.config.get(Constants.MODEL)),
-                    api_key=str(self.config.get(Constants.API_KEY)),
-                    api_type=Constants.API_TYPE,
-                    temperature=0,
-                )
+            num_output = (
+                int(self.config.get(Constants.NUM_OUTPUT, 50))
+                if self.config.get(Constants.NUM_OUTPUT) is not None
+                else None
+            )
```

Comment:
Can you elaborate on what we're trying to do here?

Reply:
num_output is optional both in UI and Google Palm. When not passed, it is set to None in Palm. I want to send either the value fed by the user (if at all this property was set) else preserve it as None as Google Palm does internally.

Reply:
In that case, this should do the job right?
```suggestion
            num_output = self.config.get(Constants.NUM_OUTPUT, None)
```

Reply:
But don't we want the int value? Won't python throw error if we do
int(self.config.get(Constants.NUM_OUTPUT, None))

Reply:
Hmm yes you're right about that. In this case its okay to have the below as well
```
num_output = (
	      int(self.config.get(Constants.NUM_OUTPUT))
	      if self.config.get(Constants.NUM_OUTPUT)
	      else None
	  )
```
1. The default value of 50 will never be taken so its unnecessary to mention that
1. The `if` check can be simplified a bit
![image](https://github.com/Zipstack/unstract-adapters/assets/117059509/793a0890-afb9-4994-81f1-85e70a8e3121)

Reply:
I remember mypy raises an error when we do not set a backup incase config.get() fails as we are passing that to int() here. Hence I added that value.